pkg/controller.v1/mpi/mpijob_controller_test.go

```diff
@@ -615,36 +615,6 @@ var _ = Describe("MPIJob controller", func() {
 		})
 	})
 
-	Context("MPIJob with ServiceAccount not controlled by itself", func() {
```

Comment:
Why do we need to return this unit test ?

Reply:
This test is related to the functionality, that a service account used and not created by the Training Operator should not be accepted. The functionality is removed with mpijob_controller.go#754, so the test also needs to be removed, since it fails otherwise

Reply:
Make sense, @rpemsel maybe we could refactor this test to verify if user creates custom ServiceAccount, MPIJob can be properly created with that service account ?

Reply:
Hi @andreyvelich,

sorry for the delay. I initially misinterpreted your comment, waiting for action by the maintainers of the repository. I meanwhile provided a new test, that checks that a provided service account is being used in a scheduled launcher.

Reply:
Thank you @rpemsel!

Reply:
I'm ok.